### ISSUE TITLE
adapt-contrib-pageLevelProgress: _showPageCompletion default value

### ISSUE THREAD

**Issue by danielghost** [adaptlearning/adapt_framework#1327]
Would suggest the default is changed to true. 

We have had issues where unless the page was completed in one session, the PLP would look incomplete even when all the components were completed and optional components were setup accordingly. The  _showPageCompletion attribute can also resolve this display issue with the progress, but maybe that requires a separate fix anyway.

**Comment by moloko** [comment]
> we have had issues where unless the page was completed in one session, the PLP would look incomplete even when all the components were completed and optional components were setup accordingly. 

@danielghost yep that's exactly what I've been using this setting to fix, it seems to crop up on any SCORM course where there are optional components in a page.

**Comment by kineotesting** [comment]
We (Kineo QA) keep running into a PLP bug where the indicator doesn’t fill in for a page if you complete it in two sessions – eg do half the components on a page, exit, re-enter, carry on – PLP for that page will only fill in 90% despite all the individual elements filling in. This usually rights itself by going to the menu (where applicable) and returning to the page

**Comment by chris-steele** [comment]
@moloko @kineotesting @danielghost this has to be because PLP monitors `_isInteractionComplete` when it should be monitoring `_isComplete`. Tracking data restores `_isComplete` properly for all blocks (and by extension) components, but not `_isInteractionComplete` (this is only handled for questions when `_shouldStoreResponses` is `true`. Simple solution that works is to switch the property. Does anyone know of any reason not to do this? N.B. this is reminiscent of the trickle issue with `_completionAttribute`.

**Comment by oliverfoster** [comment]
+1 to flipping the attribute as default

**Comment by chris-steele** [comment]
@oliverfoster sorry didn't mean to exclude you from the @ I'm almost hesistant about this at is seems too obvious to have been missed, but it looks more glaring the more I look at it.

**Comment by oliverfoster** [comment]
I completely agree. PLP shows course completion. It should be extended to do assessment completion properly, but using ```isInteractionsComplete``` here clearly isn't the right way to show that other things need to be done.... Although it might be for assessment questions, but we can cross that bridge when we integrate assessments properly.

**Comment by moloko** [comment]
now that https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/pull/93 has been merged, do we think any further action on this is still required? Is the `_showPageCompletion` still relevant at all?

**Comment by oliverfoster** [comment]
if we turn this off we still need to find a way to flag to the user and the course creator that mandatory components which are excluded from the PLP need to be completed.
